example/src/App.tsx

```diff
@@ -49,13 +68,13 @@ class App extends React.Component {
     }});
 
     Radar.getLocationPermissionsStatus().then((result) => {
-      alert(JSON.stringify(result));
+      this.logOutput(JSON.stringify(result));
```

Comment:
Can you add calls to `App.tsx` for all of the new functions and all of the new params for existing functions?

Reply:
OK, I will add all the new functions here